dbos/_dbos_config.py

```diff
@@ -301,6 +311,17 @@ def load_config(
 
     """
 
+    # Get the current call stack
+    frame_records = inspect.stack()
+    called_from_cli = False
+    for frame_record in frame_records:
+        if "dbos/cli/cli.py" in frame_record.filename:
+            called_from_cli = True
+    if not called_from_cli:
+        dbos_logger.warning(
+            "This function is deprecated and will be removed in a future version."
```

Comment:
Add some instructions like, "This will be deprecated in favor of DBOSConfig in an upcoming release."

Reply:
good idea, done